hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -0,0 +1,115 @@
+# Pi4 Bootloader Configuration
+
+You can display the current configuration using 
```

Comment:
@JamesH65 Maybe it's worth squeezing in an introductory sentence here, before diving straight into "You can display the current configuration" ?  :man_shrugging: (e.g. something about the bootloader config being stored inside the EEPROM image?)

Reply:
To be precise "vcgencmd bootloader_config" returns the configuration used by the bootloader at bootup. There's no way of knowing what the 'current' config is if e.g. someone has updated it via flashrom without a reboot OR put a new pieeprom.upd in /boot

Reply:
What if the first paragraph was changed to "You can display the currently-active configuration using" and then the next paragraph was amended to mention that you need to restart the Pi for the changes to take affect?

Reply:
Done.